packages/experiments/src/components/Coachmark/Coachmark.tsx

```diff
@@ -0,0 +1,226 @@
+// Utilities
+import * as React from 'react';
+import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
+
+// Component Dependencies
+import { PositioningContainer } from '../PositioningContainer/PositioningContainer';
+
+// Coachmark
+import { ICoachmarkProps } from './Coachmark.Props';
+import { getStyles } from './Coachmark.Styles';
+
+/**
+ * An interface for the cached dimensions of entity inner host.
+ */
+export interface IEntityRect {
+  width: number,
```

Comment:
It seems like we already have a rect interface. is this redunant? Maybe search to be sure.